templates/benchmark_name_benchmark_pipeline.py.template

```diff
@@ -0,0 +1,31 @@
+from mcs_benchmark_data._pipeline import _Pipeline
```

Comment:
You can put the templates in the source tree, like `mcs_benchmark_data.cli.templates` or similar.